build-aci

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 tag" >/dev/stderr
+    exit 1
+fi
+
+tag=$1
+
+tgt=$(mktemp -d)
+
+# Build fleet inside
+docker run -v $PWD:/opt/fleet google/golang:1.3.1 /bin/bash -c "cd /opt/fleet && ./build"
+
+# Generate manifest into target tmp dir
+cat <<DF >${tgt}/manifest
+{
+   "acVersion" : "0.5.1",
```

Comment:
Maybe `actool build` can check the acVersion in the manifest, change it to the last version and:
- either update other fields if there are incompatibilities between the last version and the specified version
- or complain (or print a warning?) that the manifest is too old: "acVersion is too told. Please fix your manifest."